### PR TITLE
Improvement/aktuelles

### PR DESCRIPTION
**Pull request by LisaFremdt**

### CODE REVIEW

**Comment by TomasKostadinov** [comment]
@BlueEnni can you compress the new images here?
Thanks!

File: site/aktuelles/2019-07-schuhtestaktion/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <div class="container">
+            <div class="hero-inner">
+                <div class="hero-card">
+                    <h1>Juli 2019: <br>Schuhtestaktion</h1> </br>
+                    <a class="button button--solid" href="/assets/pdf/2019_04_24FlyerLaufenWalkingA4Schier.pdf">Zur PDF-Datei</a>
+                </div>
+            </div>
+        </div>
+    </div>
+<div class="container distance">
```
Comment (TomasKostadinov) [inline site/aktuelles/2019-07-schuhtestaktion/index.html]: This page is not needed, can you delete it and link directly to the pdf file?

File: site/aktuelles/2019-05-trail-running-festival-innsbruck/index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+                <h1> Mai 2019: <br> Trailrunning Festival</h1>
+                <p>Bilder hierzu finden Sie in unserer Bildergalerie. </p>
+                <a class="button button--solid" href="/bilder/2019-05-trail-running-festival-innsbruck">Zur Bildergalerie</a>
+            </div>
+        </div>
+    </div>
+</div>
+<div class="container distance">
+
+    <h2></h2>
```
Comment (TomasKostadinov) [inline site/aktuelles/2019-05-trail-running-festival-innsbruck/index.html]: Delete this (not needed, is empty)

File: site/aktuelles/2019-07-maintal-ultra-marathon/index.html [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                    <p>Bilder hierzu finden Sie in unserer Bildergalerie. </p>
+                    <a class="button button--solid" href="/bilder/2019-07-maintal-ultra-marathon">Zur Bildergalerie</a>
+                </div>
+            </div>
+        </div>
+    </div>
+<div class="container distance">
+
+    <p>
+        <h2>Bericht zum 6. Maintal Ultra Marathon in Veitshöchheim / Sa. 13.07.2019</h2>
```
Comment (TomasKostadinov) [inline site/aktuelles/2019-07-maintal-ultra-marathon/index.html:18]: p -> h2 is wrong markup.
please remove the p tags

File: site/aktuelles/2019-07-maintal-ultra-marathon/index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+   </p>
+
+   <p>
+        <h4>Ulrike Kinzler erfolgreich beim Maintal Ultratrail</h4>
```
Comment (TomasKostadinov) [inline site/aktuelles/2019-07-maintal-ultra-marathon/index.html:22]: same here

File: site/aktuelles/index.html [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
+    
+                        </div>
+                    </div>
+                </a>
+               <a>
+
+               </a>
+               <a>
+                   
+               </a>
```
Comment (TomasKostadinov) [inline site/aktuelles/index.html]: Not needed

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]
Looks good otherwise!

**Review by TomasKostadinov — CHANGES_REQUESTED** [review]
Looks very good @LisaFremdt, very proud 😊!

Now @BlueEnni needs to compress the images and we can merge this.

**Comment by BlueEnni** [comment]
> Looks very good @LisaFremdt, very proud 😊!
> 
> Now @BlueEnni needs to compress the images and we can merge this.

All done - merging may be possible now :)

**Review by BlueEnni — APPROVED** [review]

**Pull request merged**